verifier/README.md

```diff
@@ -0,0 +1,8 @@
+[![img](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://www.quebec.ca/gouv/politiques-orientations/vitrine-numeriqc/accompagnement-des-organismes-publics/demarche-conception-services-numeriques)
```

Comment:
It looks like this does not run out of the box with the rest of the demo.  I assume we would just need to add the dockerfile contents into the docker-compose.yml file.

I was thinking that the verifier would be a "here's how to use your new credential" capability for the user, so it was part of the existing issuer UI.  Totally optional, but would let a user experience the presentation side of the experience.  If we did do that, I think we need:

* add flow that gets them start from the existing screen
* to add more information to the user about what they are doing
* minimize the extra steps -- e.g. not have them pick flights -- and focus just on the scanning and presentation process

Alternatively, this could be just a separate repo that verifier's could use as a reference for building their own verifier.

What to you think about that?

Reply:
All right, I am going to discuss with the team a way to conform with these needs. 

As I gather, the "user story" booking a flight isn't necessary here. We can go from issuing to consuming the scanning without doing much else? 

If we link the issuing page to the consuming so the user can present his credential and he's told what's going on under the hood, will it be good enough? 

Tks!

Reply:
Yes -- that would be great.

As a standalone demo of the verifier, it is excellent, so it might be worth keeping it as a separate repo.  But for integrating with the issuer, I think we should go to a reduced story.

Reply:
Ok Stephen, we're going to adapt it. I also sent a msg to @flihp27 about the license, and as soon as he is available, I am going to talk about the licensing issue you raised. Be back to you soon. 

Tks!